src/gui/widgets/EffectView.cpp

```diff
@@ -97,9 +97,13 @@ EffectView::EffectView( Effect * _model, QWidget * _parent ) :
 		if( m_controlView )
 		{
 			m_subWindow = gui->mainWindow()->addWindowedWidget( m_controlView );
-			m_subWindow->setSizePolicy( QSizePolicy::Fixed, QSizePolicy::Fixed );
-			if (m_subWindow->layout()) {
-				m_subWindow->layout()->setSizeConstraint(QLayout::SetFixedSize);
+
+			if ( !m_controlView->isResizable() )
+			{
+				m_subWindow->setSizePolicy( QSizePolicy::Fixed, QSizePolicy::Fixed );
+				if (m_subWindow->layout()) {
+					m_subWindow->layout()->setSizeConstraint(QLayout::SetFixedSize);
+				}
```

Comment:
I didn't; since this is my first big contribution, I tried to limit any changes outside the plugin folder itself to the minimum. Allowing maximize when the window is resizable seems like a good idea though, I'll try it and see if it breaks anything.

Reply:
So, supporting maximization will probably require some more thought -- it basically works, but the title bar / window decoration does not disappear when the window is maximized and controls are drawn over it. So I would rather leave it for a later feature update.
Also, it would make it tempting to maximize the analyzer just because it can be done, which may not be a great idea -- certain types of spectra (mainly noise) have quite heavy performance demands, and the larger the window is, the more time it takes to draw them.
I even considered limiting the maximum size to some "reasonable" value, but what's reasonable will differ based on screen size, its resolution, how fast hardware the user has etc., so I left it "unlocked" in the end.

Reply:
That's fine then :+1: